src/Redux/Slices/usersSlice.js

```diff
@@ -60,11 +60,11 @@ const usersSlice = createSlice({
       .addCase(createUser.fulfilled, (state, action) => {
         state.loading = false;
 
-        if (action.payload?.errors) {
+        if (typeof action.payload === 'string') {
           state.status = 'Failed to sign up';
+        } else {
+          state.status = 'created';
         }
```

Comment:
@AndreaM2429 Once the user has registered, it's redirected to the Main page, consider saving the user's information in the store so that they can keep their session logged in.